crates/cast/tests/helpers/fixtures.rs

```diff
@@ -68,7 +74,24 @@ pub async fn declare_deploy_contract(account: &str, path: &str) {
 
     let factory = ContractFactory::new(class_hash, &account);
     let deployment = factory.deploy(Vec::new(), FieldElement::ONE, true);
-    deployment.send().await.unwrap();
+
+    let transaction_hash = deployment.send().await.unwrap().transaction_hash;
+    let receipt = get_transaction_receipt(transaction_hash).await;
+    let mut address = None;
+    match receipt {
+        TransactionReceipt::Invoke(invoke_receipt) => {
+            for event in invoke_receipt.events {
+                if event.from_address == FieldElement::from_hex_be(UDC_ADDRESS).unwrap() {
+                    address = event.data.first().copied();
+                    break;
+                }
+            }
+        }
+        _ => {
+            panic!("Unexpected TransactionReceipt variant");
+        }
+    };
```

Comment:
Wouldn't it be more straight forward to just calculate the address ourselves using `get_contract_address`?

Reply:
It probably would, but I actually wanted to avoid depending on this. No specific reasons, I just thought it may be more lightweight and overall a better idea to just get the values straight from devnet.